pipelines/simple_elt/main.py

```diff
@@ -311,5 +314,96 @@ def merge_gerrydb_tilesets(
     )
 
 
+@cli.command("load-districtr-v1-places")
+@click.option("--replace", is_flag=True, help="Replace existing files", default=False)
+def load_districtr_v1_places(replace: bool = False) -> None:
+    """
+    Load data from districtr_v1 endpoint to the s3 bucket for later ingestion
+    """
+    districtr_places = urlretrieve(
+        "https://districtr.org/assets/data/landing_pages.json?v=2",
+        settings.OUT_SCRATCH / "districtr_v1_places.json",
+    )
+
+    s3_client = settings.get_s3_client()
+
+    key = f"{S3_PREFIX}/districtr_places/districtr_v1_places.json"
+    print(s3_client, settings.S3_BUCKET, key)
+    s3_client.upload_file(districtr_places, settings.S3_BUCKET, key)
+
+
+@cli.command("load-districtr-v1-problems")
+@click.option("--replace", is_flag=True, help="Replace existing files", default=False)
+def load_districtr_v1_problems(replace: bool = False) -> None:
+    """
+    load problems definition json file for states from districtr_v1 and store in s3 bucket
+    """
+    s3_client = settings.get_s3_client()
+
+    # check if the districtr_places object exists in s3; if not, download it using load_districtr_v1_places
+    key = f"{S3_PREFIX}/districtr_places/districtr_v1_places.json"
+    if not exists_in_s3(s3_client, settings.S3_BUCKET, key):
+        load_districtr_v1_places()
+
+    districtr_places = download_file_from_s3(s3_client, urlparse(key), replace)
+
+    with open(districtr_places, "r") as file:
+        places = json.load(file)
+
+    for place in places:
+        url = place["state"]
+        LOGGER.info(f"Downloading problems for {url}")
+        try:
+            problems = urlretrieve(
+                f"https://districtr.org/assets/data/modules/{place['state'].lower()}.json",
+                settings.OUT_SCRATCH / f"{place['state'].lower()}_problems.json",
+            )
+            key = (
+                f"{S3_PREFIX}/districtr_problems/{place['state'].lower()}_problems.json"
+            )
+            s3_client.upload_file(problems, settings.S3_BUCKET, key)
+        except Exception as e:
+            LOGGER.error(f"Failed to download problems for {url}: {e}")
+            continue
+
+
 if __name__ == "__main__":
     cli()
+
+
+def upsert_places_and_problems():
```

Comment:
PP: Move above `if __name__ == "__main__"` but add `raise NotImplementedError` in the function body. Or just `warnings.warn("Not implemented")` or something